src/ProgressOnderwijsUtils/Data/ParameterizedSql.cs

```diff
@@ -310,6 +317,12 @@ static ParamRefSubString ParamRefNextMatch(string query, int pos, int length)
         while (pos < length) {
             var c = query[pos];
             if (c == '{') {
+                if (pos + 1 > length) {
+                    break;
+                } else if (query[pos + 1] == '{') {
+                    pos += 2;
+                    continue;
+                }
```

Comment:
Dit had ik gedaan om een eerste poging te hebben om `{` in SQL te ondersteunen, maar bij nader inzien is dat niet de handigste strategie.

```suggestion
```